code/src/components/app-input/AppInput.scss

```diff
@@ -1,12 +1,33 @@
 @import "@design-system";
 
-.spa-app-input {
-  &__textfield {
-    .MuiInputBase-input {
-      padding: spacing(3) spacing(4);
-    }
-    .MuiFormLabel-root {
-      padding-top: 3px;
+$spa-input-colors: (
+  "contained": var(--spa-input-contained-color, $purple),
+  "success": var(--spa-input-success-color, $green),
+  "danger": var(--spa-input-danger-color, $red),
+  "light": var(--spa-input-light-color, $white),
+  "dark": var(--spa-input-dark-color, $black)
+);
```

Comment:
maybe create these variables like general for whole project ? 
These items also used for `AppSelect` maybe create some `common` and reuse it ? 
What do you think ?

Reply:
We already using global variables. Also, as you can see we are using different names for that. For example, here are `--spa-input-color`, in the app-select `--spa-select-color`.